grisera_api/scenario/scenario_service_graphdb.py

```diff
@@ -18,8 +18,10 @@ class ScenarioServiceGraphDB(ScenarioService):
     experiment_service (ExperimentService): Service used to communicate with Experiment
     """
     graph_api_service = GraphApiService()
-    activity_execution_service = ActivityExecutionServiceGraphDB()
-    experiment_service = ExperimentServiceGraphDB()
+
+    def __init__(self):
+        self.activity_execution_service = None
+        self.experiment_service = None
```

Comment:
Missing type definition of these services

Reply:
As above